src/bun.js/webcore/streams.zig

```diff
@@ -2592,8 +2592,8 @@ pub fn HTTPServerWritable(comptime ssl: bool) type {
         // so it must zero out state instead of make it
         pub fn finalize(this: *@This()) void {
             log("finalize()", .{});
-
             if (!this.done) {
+                _ = this.flushFromJSNoWait();
```

Comment:
What prevents this from executing JS?

Reply:
Fixed here: https://github.com/oven-sh/bun/pull/15324/commits/94e36557bebea7c948e58f172f3d774f91fcc846